pump/main/models.py

```diff
@@ -104,6 +127,17 @@ def percentage_positive(self):
         percents = ['4.4', '6.1', '7.9', '18.6', '40.7']
         return percents[self.number_passed()]
 
+    def fall_related_injury(self):
```

Comment:
Hey, I'm just curious - where is this functionality defined? The `over_55()` function makes sense but it's not really obvious why `fall_related_injury()` depends on the user's sex, race, etc.

Reply:
That's a good question.  Its defined by a word doc the faculty member sent us, which I suppose makes the definitions below magic-numbery.  Should I add some comments to explain?

Reply:
Okay - yeah, that's a good idea. I would add a comment to this function saying that this is defined in that word doc, because from first glance it looks like it could be a mistake or oversight or something.